quick_start/workflow.rst

```diff
@@ -72,4 +137,4 @@ And if we look at the log of our process, which we configured as the sink, we wi
  > nstack log 1
  Feb 17 09:59:26 nostromo nstack-server[8925]: OUTPUT: 3
 
-Great!
+Great - we can see that the output of our function (and the number of characters in "Foo") is 3.
```

Comment:
I think we need to move Advanced: Framework Modules to the Advanced tutorial shortly. We could put it there.